src/style-auro-ld.scss

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// general WCSS imports
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/utilityMixins/focusVisible";
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/animation";
+
+// Import Auro SCSS variables
+@import "./node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SCSSVariables";
+
+// import button styles into element namespaced wrapper
+auro-button-light {
+  @import './style-auro.scss';
+}
+
+// light DOM support for hover states
+auro-button-light:not(.is-touching) {
+  .auro-button {
+    &:hover {
+      @include auro_transition(all, 150ms, ease-in-out);
+      
+      background-color: var(--auro-color-ui-hover-on-light);
```

Comment:
It's bugging me that these are duplicated from the main stylesheet. This is why I had to make all those hover states mixins with ods-button. 

I really want to find a better way to manage this interaction between shadow and light DOM elements.

Reply:
We briefly discussed this problem a couple days ago, but I'm having trouble remembering our conclusion. Should we move back towards the hover states mixin pattern? What would you propose aside from the mixins to solve this problem?

Reply:
I am going to resolve this conversation as I don't have an answer right now. But this is something that needs to be refactored at a future date.